homeassistant/components/vivotek/camera.py

```diff
@@ -98,7 +98,11 @@ def camera_image(
         self, width: int | None = None, height: int | None = None
     ) -> bytes | None:
         """Return bytes of camera image."""
-        return self._cam.snapshot()
+        snapshot = self._cam.snapshot()
+        if type(snapshot) is not bytes:
+            return None
+
+        return snapshot
```

Comment:
You're probably doing this because mypy was complaining you shouldn't return Any? 

Well that is because the library is likely lacking a `py.typed` file. Without that file, mypy can't read the types from the library. So unless there is a real reason why snapshot would not return bytes or None, we shouldn't have this :)